### DISCUSSION TITLE
Help matching transactions between two accounts as transfers

### DISCUSSION THREAD

**Discussion by Inrego** [firefly-iii/firefly-iii#7886] [Firefly III]
I'm having issues correctly marking transactions as transfers between two accounts. I'm importing with Nordigen (GoCardless).
For example, I would like to match these two transactions:

            {
                "transactionId": "P9023294122202308010000000167",
                "bookingDate": "2023-08-01",
                "valueDate": "2023-08-01",
                "transactionAmount": {
                    "amount": "7000.00",
                    "currency": "DKK"
                },
                "remittanceInformationUnstructured": "Bgs El",
                "remittanceInformationStructured": "El",
                "additionalInformation": "BGS",
                "internalTransactionId": "0f92d2c806490acba869ed3e33066a85"
            },

and 

            {
                "transactionId": "P9023293835202308010000000350",
                "bookingDate": "2023-08-01",
                "valueDate": "2023-08-01",
                "transactionAmount": {
                    "amount": "-7000.00",
                    "currency": "DKK"
                },
                "creditorName": "2000",
                "creditorAccount": {
                    "iban": "902329xxxx"
                },
                "remittanceInformationUnstructured": "Bgs El",
                "remittanceInformationStructured": "El",
                "additionalInformation": "BGS",
                "internalTransactionId": "35bc011fb378351857bba9ff63e5b7fa"
            },

In Firefly, the transactions are added like this:

![image](https://github.com/firefly-iii/firefly-iii/assets/2242329/90e22cdc-17cd-4dd9-b08d-f9ae69ecbd53)

and

![image](https://github.com/firefly-iii/firefly-iii/assets/2242329/c37f2bfc-a026-4483-bdeb-cdf9f6036ab7)

The `902329xxxx` in `creditorAccount:iban` is actually the account number (which is also the last 10 digits of the IBAN, but also entered as Account number on that account).

Any help on how to correctly import these transactions?

**Comment by JC5** [comment]
Contact Nordigen, that field should not contain that number as the "iban". Firefly III will not accept invalid IBANs so the field will always be dropped.

**Reply by Inrego** [reply]
Alright - so if I get them to fix that. Will they then be marked correctly as a transfer? Seeing as the other transaction doesn't have any information about the source account.

**Reply by JC5** [reply]
I can't be sure. The importer should fall back on the default account but this is new for me too.
